Userland/Applications/HexEditor/HexEditorWidget.cpp

```diff
@@ -610,3 +621,25 @@ void HexEditorWidget::drop_event(GUI::DropEvent& event)
         open_file(response.value().filename(), response.value().release_stream());
     }
 }
+
+ErrorOr<String> HexEditorWidget::make_display_string(StringView const& string)
```

Comment:
Always pass StringView by value (https://github.com/SerenityOS/serenity/commit/8b1108e4858f797c9216dc8ae4a3918ad50c73b4 for reference)